### PR TITLE
Add Synthetics blog link

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Add Synthetics blog link

### Motivation
New blog post

### Preview
https://docs-staging.datadoghq.com/ruth/docs-blog-link-7-22/synthetics/guide/app-that-requires-login

### Additional Notes
N/A

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by alai97 — APPROVED** [review]
The blog link looks good to me! Quick question on the second and third links: should we make these sentence case instead of title case for "Browser Tests?"

File: content/en/synthetics/guide/app-that-requires-login.md [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
-
+  - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+    tag: 'Blog'
+    text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+  - link: 'synthetics/browser_tests'
+    tag: 'Documentation'
+    text: 'Configure a Browser Test'
+  - link: '/synthetics/browser_tests/actions'
+    tag: 'Documentation'
+    text: 'Create Browser Test Steps'
```
Comment (alai97) [inline content/en/synthetics/guide/app-that-requires-login.md:13]:
```suggestion
    text: 'Configure a browser test'
  - link: '/synthetics/browser_tests/actions'
    tag: 'Documentation'
    text: 'Create browser test steps'
```

Comment (ruthnaebeck) [reply content/en/synthetics/guide/app-that-requires-login.md:13]: Personally, I'd rather the text be the page title (title case). I will update.

File: content/en/synthetics/guide/app-that-requires-login.md [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
-    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
-      tag: 'Blog'
-      text: 'Best practices for creating end-to-end tests'
-
+  - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+    tag: 'Blog'
+    text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+  - link: 'synthetics/browser_tests'
+    tag: 'Documentation'
+    text: 'Configure a Browser Test'
```
Comment (ruthnaebeck) [inline content/en/synthetics/guide/app-that-requires-login.md:10]:
```suggestion
    text: 'Browser Tests'
```

File: content/en/synthetics/guide/app-that-requires-login.md (hunk 1 above)
Comment (ruthnaebeck) [reply content/en/synthetics/guide/app-that-requires-login.md:13]:
```suggestion
    text: 'Browser Test Steps'
```

**Pull request merged**